src/Components/Search/Style.js

```diff
@@ -0,0 +1,43 @@
+import styled from "styled-components";
+
+export const SearchItem = styled.div`
+  display: flex;
+  width: 384px;
+  justify-content: space-between;
+  align-items: center;
+  font-size: 22px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 30px;
+  letter-spacing: -0.22px;
+`;
+export const SearchItemTitle = styled.div`
+  display: flex;
+  align-items: center;
+  gap: 8px;
+  &:first-child {
+    gap: 16px;
+  }
+`;
+export const SearchBox = styled.div`
+  display: grid;
+  gap: 16px;
+  box-sizing: border-box;
+  width: 448px;
+  backgroundcolor: #999;
```

Comment:
background-color로 수정하시는게 좋을 것 같아요.

```suggestion
  background-color: #999;
```